src/components/DocSigningField.vue

```diff
@@ -0,0 +1,78 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<p>
+		<label :for="id">
+			{{ label }}
+		</label>
+		<br>
+		<textarea :id="id"
+			v-model="localValue"
+			:name="name" />
+		<br>
```

Comment:
You can use [NcTextArea](https://nextcloud-vue-components.netlify.app/#/Components/NcFields?id=nctextarea) which comes with proper styling for the label

Reply:
Done